plugins/coverbrowser.koplugin/main.lua

```diff
@@ -132,15 +133,43 @@ function CoverBrowser:addToMainMenu(menu_items)
             separator = true,
         },
         {
-            text = _("Shorten home directory to Home"),
+            text = _("Shorten home directory"),
             checked_func = function() return G_reader_settings:readSetting("home_dir_display_name") end,
             callback = function()
                 if G_reader_settings:readSetting("home_dir_display_name") then
-                    G_reader_settings:delSetting("home_dir_display_name")
-                    if FileManager.instance then FileManager.instance:reinit() end
+                  G_reader_settings:delSetting("home_dir_display_name")
+                  if FileManager.instance then FileManager.instance:reinit() end
                 else
-                    G_reader_settings:saveSetting("home_dir_display_name", _("Home"))
-                    if FileManager.instance then FileManager.instance:reinit() end
+                  local home_path_dialog
+                  home_path_dialog = InputDialog:new{
+                    title = _("Home directory name"),
```

Comment:
Also generic, the indentation seems to have ended up as two spaces instead of four.